.yamllint

```diff
@@ -1,6 +1,6 @@
----
 extends: default
 rules:
+  document-start: disable
```

Comment:
Note: this is not because I don't like it but because the yaml autoformatter doesn't yet support it:
https://github.com/macisamuele/language-formatters-pre-commit-hooks/blob/c4eae5ea2cd9a098fba0db73ab6f5b02777835b2/language_formatters_pre_commit_hooks/pretty_format_yaml.py#L46-L63